south-african-journal-of-geology.csl

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>South African Journal of Geology</title>
+    <title-short>Geol. South Africa</title-short>
+    <id>http://www.zotero.org/styles/south-african-journal-of-geology</id>
+    <link rel="self" href="http://www.zotero.org/styles/south-african-journal-of-geology"/>
+    <link href="http://www.gssa.org.za/?page_id=108" rel="documentation"/>
+    <author>
+      <name>Shaun Murdoch</name>
+      <email>shaun.murdoch@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="geology"/>
+    <issn>1012-0750</issn>
+    <eissn>1996-8590</eissn>
+    <summary>Style for the South African Journal of Geology.</summary>
+    <updated>2016-07-07T12:07:56+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
+    <terms>
+      <term name="editor" form="short">
+        <single>Editor</single>
+        <multiple>Editors</multiple>
```

Comment:
The only reason for doing what you're doing would be if you'd want "long" non-abbreviated terms for "editor", but "short" abbreviated terms for "translator" when editors and translators are substituted in the "author" and "author-short" macros.